fdbclient/SystemData.cpp

```diff
@@ -1410,6 +1410,16 @@ const KeyRef tenantMapPrivatePrefix = "\xff\xff/tenantMap/"_sr;
 const KeyRef tenantLastIdKey = "\xff/tenantLastId/"_sr;
 const KeyRef tenantDataPrefixKey = "\xff/tenantDataPrefix"_sr;
 
+const KeyRangeRef storageQuotaKeys(LiteralStringRef("\xff/storageQuota/"), LiteralStringRef("\xff/storageQuota0"));
+const KeyRef storageQuotaPrefix = storageQuotaKeys.begin;
```

Comment:
Is this something we should store as part of the tenant metadata? I think a possible litmus test for that is whether or not this information needs to be relocated with a tenant when we eventually support tenant movement and/or whether the management cluster would have any need to know this information.

Reply:
This will be changed to be quota for capacity/tenant groups (once the tenant groups functionality has been added). Is it possible that a tenant group could contain tenants that span multiple clusters? If so, I suspect the quota information will need to be in the management cluster. Otherwise, it could move with the tenant groups. What do you think?

Reply:
The tenant groups will be restricted to a single cluster, so then it probably makes sense to track this data as metadata on the tenant group. I guess in the interim, then, we probably shouldn't add it directly to the tenants.

Reply:
That sounds good to me